package.json

```
@@ -34,6 +34,7 @@
   },
   "homepage": "https://github.com/silvermine/videojs-chromecast#readme",
   "dependencies": {
+    "sass": "^1.52.2",
```

Just checking, should this be a dev dependency? Also:

```suggestion
    "sass": "1.52.2",
```

---

Is this still supposed to be a dependency?

---

Sorry about that. Did not notice it was not removed as a dependency when reinstalling with `--save-dev`. Should be good to go now.